integration-cli/utils.go

```diff
@@ -83,3 +84,26 @@ func parseCgroupPaths(procCgroupData string) map[string]string {
 func runAtDifferentDate(date time.Time, block func()) {
 	integration.RunAtDifferentDate(date, block)
 }
+
+func findCgroupMountpoints() map[string]string {
+	cgMounts, err := cgroups.GetCgroupMounts()
```

Comment:
> Well, isn't that also the case for "UserNamespaceInKernel" in requirements.go?

Ah... got it: UserNamespaceInKernel is always specified together with "SameHostDaemon".